lib/tarpon/request/subscriber.rb

```diff
@@ -3,8 +3,8 @@
 module Tarpon
   module Request
     class Subscriber < Base
-      def initialize(app_user_id:)
-        super()
+      def initialize(app_user_id:, **opts)
```

Comment:
As it is required can we require `client` argument explicitly here?
```suggestion
      def initialize(app_user_id:, client:)
```